wwan/app/quectel-cm/files/quectel.sh

```diff
@@ -36,6 +36,7 @@ proto_quectel_setup() {
 	json_get_vars pdptype dhcp dhcpv6 sourcefilter delegate ip4table
 	json_get_vars ip6table mtu $PROTO_DEFAULT_OPTIONS
 
+	echo -ne "AT+CFUN=1\r\n" > /dev/ttyUSB2
```

Comment:
Could you add a comment block describing what this is doing?

Something like
# Set Phone Functionality as Full (with small description up top)